integrations/pinecone/tests/test_embedding_retriever.py

```diff
@@ -174,3 +174,12 @@ def test_run():
     assert len(res["documents"]) == 1
     assert res["documents"][0].content == "Test doc"
     assert res["documents"][0].embedding == [0.1, 0.2]
+
+
+async def test_run_async():
```

Comment:
```suggestion
@pytest.mark.asyncio
async def test_run_async():
```

from logs
> tests/test_embedding_retriever.py::test_run_async
... PytestUnhandledCoroutineWarning: async def functions are not natively supported and have been skipped.

Reply:
Thanks for very helpful review.